contrib/runners/mistral_v2/query/mistral_v2.py

```diff
@@ -155,12 +171,14 @@ def _get_workflow_tasks(self, exec_id, last_query_time=None):
         result = []
 
         try:
-            query_filters = {}
+            wf_tasks = self._client.tasks.list(workflow_execution_id=exec_id)
```

Comment:
How is expensive is this now that we don't so filtering server-side (aka on Mistral API) anymore?

It there any pagination or something else we need to be careful of? Or the client library takes care of that?